1208-projects/project_landing_page.feature

```diff
@@ -2,26 +2,55 @@ Feature: Project Landing Page
 
     @2630
     Scenario: User opens Landing Page for Project
-        Given Anonymous User views Landing Page for Registration
-        And the Registration has a Project
-        When they click the Link to a Project
+        Given A Anonymous User uses a browser
+        When they open a Project Landing Page
```

Comment:
syns kanskje den forrige Given-And-When hadde en bedre beskrivelse for hvordan man faktisk åpner et prosjekt

Reply:
Vi veit ikkje kvifor eller korleis brukar opnar ein landing page. 
Det er poenget med ein landing page, den må klare å stå på egne bein.
Forutsetninga om eksistens av ein Registrering (av ein publikasjon) er feil. 
Prosjekt utan publikasjonar, har også landing page.

Reply:
Sant nok. Det er bare at `Given A Anonymous User uses a browser` får meg til å tenke at brukeren også trenger PC/mobil, strøm, internett, være våken, være født, osv, så hvor langt skal vi dra Given :p

I dette tilfellet har ikke Given noe med NVA å gjøre, men jeg ser heller ikke noe bedre løsning, så hvis @Eiriknil ikke ser noe problem med det er det får det være greit nok :)

Reply:
Eg ser poenget ditt, men også ein begynnelse - må starte med eit sannferdig utgangspunkt.

Reply:
Skal ikke være noe problem med det her.